charts/kured/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.6.1"
```

Comment:
Should the `appVersion` changed too?

Reply:
No not now. This will be done when we make the new kured release.

Reply:
As you mentioned: This has to wait for the 1.7.0 release to merge, so maybe we can bump it to "1.7.0" already, though it will certainly fail until 1.7.0 is public. Which seems fair to me. :)

Reply:
Done.